src/loader-recorder-v2.ts

```diff
@@ -365,6 +366,43 @@ function prepareRequest(
     return requests
 }
 
+function _checkForCannotReadBody(res: Response): string | null {
+    if (res.headers.get('Transfer-Encoding') === 'chunked') {
+        return 'Chunked Transfer-Encoding is not supported'
+    }
+
+    const contentType = res.headers.get('Content-Type')
+    if (contentType && (contentType.startsWith('video/') || contentType.startsWith('audio/'))) {
+        return `Content-Type ${contentType} is not supported`
+    }
+
+    if (!res.headers.has('Content-Length')) {
```

Comment:
is `headers.get` case insensitive? if not, let's make it since headers are not guaranteed to be always capitalized.

Reply:
`headers.get` is case insensitive https://developer.mozilla.org/en-US/docs/Web/API/Headers/get#name
the spec doesn't state for `has` https://developer.mozilla.org/en-US/docs/Web/API/Headers/has

but it is case insensitive
<img width="345" alt="Screenshot 2024-01-31 at 10 40 42" src="https://github.com/PostHog/posthog-js/assets/984817/4ed664df-4fc2-4755-9c9e-f531d69bce9f">

it returns the header value with the casing that was supplied
<img width="278" alt="Screenshot 2024-01-31 at 10 41 57" src="https://github.com/PostHog/posthog-js/assets/984817/7ab3ed47-d152-4b52-beda-3125d5ba6b52">